esphome/components/rc522/rc522.h

```diff
@@ -235,6 +236,7 @@ class RC522 : public PollingComponent {
   uint8_t error_counter_ = 0;  // to reset if unresponsive
   uint8_t rx_align_;
   uint8_t *valid_bits_;
+  uint8_t gain_ = 4;
```

Comment:
Default in schema means the setter is always called
```suggestion
  uint8_t gain_;
```